docs/benchmarks.md

```diff
@@ -0,0 +1,132 @@
+# Benchmarks
+
+The `integration_test` module contains benchmarks derived from the 
+[TPC-DS](http://www.tpc.org/tpcds/), [TPC-H](http://www.tpc.org/tpch/), and 
+[TPCx-BB](http://www.tpc.org/tpcx-bb/default5.asp) benchmarks. These are not official TPC 
+benchmarks and are only intended to be used to compare relative performance between CPU and GPU 
+and to help catch performance regressions in the plugin.
+
+## Data Generation
+
+For each of these benchmarks, source data must be generated by a utility that can generate data at
+any scale factor, where the scale factor is an integer representing approximately how many
+gigabytes of data will be generated, with scale factor 1 meaning ~1 GB, and scale factor 1000 meaning ~1 TB,
+for example.
+
+Further information on data generation can be found using the following links:
+
+- [TPC-DS Data Generator](https://github.com/databricks/tpcds-kit)
+- [TPC-H Data Generator](https://github.com/electrum/tpch-dbgen)
```

Comment:
I don't know if this is the one we use for TPC-H or not. It was just the first result I saw in Google.

Reply:
@revans2 Do you know if this is the correct TPC-H generator to link to here?

Reply:
This is what I used for TPC-H.  I have also downloaded some from http://www.tpc.org/tpch/ in the past, but you have to sign up for that.